Packs/Base/Scripts/CommonServerPython/CommonServerPython_test.py

```diff
@@ -3323,6 +3323,79 @@ def test_return_results_mixed_results(mocker):
     assert demisto_results_mock.call_args_list[1][0][0] == mock_demisto_results_entry
 
 
+class TestExecuteCommand():
```

Comment:
can we also add test for the other cases in the method so we have full coverage (e.g. extract_contents=False, the response from `executeCommand` is not a dict or a list, the response from `executeCommand` is a list with one dict)?

Reply:
@yaakovi Add more tests.